datasets/ade_corpus_v2/README.md

```diff
@@ -0,0 +1,219 @@
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://www.sciencedirect.com/science/article/pii/S1532046412000615
+- **Repository:** [Needs More Information]
+- **Paper:** https://www.sciencedirect.com/science/article/pii/S1532046412000615
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** [Needs More Information]
+
+### Dataset Summary
+
+ADE-Corpus-V2  Dataset: Adverse Drug Reaction Data.
+ This is a dataset for Classification if a sentence is ADE-related (True) or not (False) and Relation Extraction between Adverse Drug Event and Drug.
+ DRUG-AE.rel provides relations between drugs and adverse effects.
+ DRUG-DOSE.rel provides relations between drugs and dosages.
+ ADE-NEG.txt provides all sentences in the ADE corpus that DO NOT contain any drug-related adverse effects.
+
+### Supported Tasks and Leaderboards
+
+Sentiment classification, Relation Extraction
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+[Needs More Information]
+
+### Data Fields
+
+{
+  "text": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "label": {
+    "feature_type": "ClassLabel",
+    "dtype": "int32",
+    "class_names": [
+      "Not-Related",
+      "Related"
+    ]
+  }
+}
+
+{
+  "text": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "drug": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "effect": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "indexes": {
+    "drug": {
+      "feature_type": "Sequence",
+      "start_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      },
+      "end_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      }
+    },
+    "effect": {
+      "feature_type": "Sequence",
+      "start_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      },
+      "end_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      }
+    }
+  }
+}
+
+{
+  "text": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "drug": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "dosage": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "indexes": {
+    "drug": {
+      "feature_type": "Sequence",
+      "start_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      },
+      "end_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      }
+    },
+    "dosage": {
+      "feature_type": "Sequence",
+      "start_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      },
+      "end_char": {
+        "feature_type": "Value",
+        "dtype": "int32"
+      }
+    }
+  }
+}
```

Comment:
```suggestion
```

Can you just add a brief description of the fields with your own words instead of this dict please ?

Reply:
yeah sure i will add it